src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md

```diff
@@ -1,18 +1,14 @@
-{% comment %}
-After the free trial details are finalized, add a link to free trial page to check for free trial eligibility.
-{% endcomment %}
-
-1. Create a [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account and log in.
-1. On the **Clusters** page, click **Create cluster**.
+1. [Create a CockroachDB {{ site.data.products.cloud }} account]({% link cockroachcloud/create-an-account.md %}). If this is your first CockroachDB Cloud organization, it will be credited with $400 in [free trial credits]({% link cockroachcloud/free-trial.md %}) to get you started.
+1. On the **Get Started** page, click **Create cluster**.
 1. On the **Select a plan** page, select **Standard**.
-1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS) in the **Cloud provider** section.
+1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS).
```

Comment:
If we avoid the parenthesis we don't need to update this when you can choose Azure.